cincinnati/src/plugins/internal/graph_builder/dkrv2_openshift_secondary_metadata_scraper/plugin.rs

```diff
@@ -276,8 +274,26 @@ impl InternalPlugin for DkrV2OpenshiftSecondaryMetadataScraperPlugin {
         })
         .await?;
 
+        let graph_data_dir = data_dir.path().to_path_buf();
         self.update_cache_state(layers, data_dir).await;
 
+        let graph_data_tar_path = self.settings.output_directory.join("graph-data.tar.gz");
```

Comment:
Wondering if we should add timestamp to the graph-data.tar.gz file?

Reply:
We might want the commit information somewhere in the tarball as it will be useful for debugging. But that cane be handled in a later PR.